onnxscript/rewriter/pattern.py

```diff
@@ -1566,6 +1567,12 @@ def _apply_to_graph_or_function(
                 if delta is None or tracer is not None:
                     continue
                 assert isinstance(delta, ReplacementSubgraph)
+                if delta.new_initializers and isinstance(graph_or_function, ir.Function):
+                    # TODO(rama): Can't add initializers to functions. But currently this is not
+                    # an issue, as we apply inlining before applying rewrite rules.
+                    continue
```

Comment:
Should we add a warning?

Reply:
I guess. An alternative would be to convert initializers into "Constant" nodes. Ok, let me do something simple for now.